src/components/constants/Global.js

```diff
@@ -0,0 +1,11 @@
+export const imgUrl = "https://image.tmdb.org/t/p/w500/";
```

Comment:
The global constants should be under source director and should available to access from anywhere in the codebase.